cases/file_to_s3_reliability/ansible/config_files/vector.toml

```diff
@@ -0,0 +1,16 @@
+data_dir = "/var/lib/vector"
+
+[sources.file]
+type = "file"
+include = ["/var/lib/verifiable-logger/output.log"]
+start_at_beginning = true
+
+[sinks.s3]
+inputs = ["file"]
+type = "aws_s3"
+region = "us-east-1"
+bucket = "file-to-s3-reliability-test-data"
```

Comment:
Again, these should be templated in _somehow_.

Reply:
I'm ok with this stuff being hard coded since it's all contained within this test. We could use variables (this is what the `configurations` folder is for in the test, but I don't think it matters too much.

Reply:
The problem is S3 bucket names have global scope, so we either have to use a different one each test run, or make this part of the state global. I think making it global is worse than templating the names.